test/integration/group_offers_export_test.rb

```diff
@@ -0,0 +1,15 @@
+require 'test_helper'
+
+class GroupOffersXlsxExportTest < ActionDispatch::IntegrationTest
+  def setup
+    @superadmin = create :user
+    10.times { create :group_offer }
+  end
+
+  test 'xlsx file is downloadable' do
+    login_as @superadmin
+    get group_offers_url(format: :xlsx)
+    assert_equal 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet',
+      response.content_type
```

Comment:
how easy is it to include in the test if the correct fields are shown in the excel file?

Reply:
Not so sure, I haven't managed it yet. Also I am not sure if it is needed, for example we don't check for the index columns either and we didn't think of testing the xlsx fields in the client and volunteers.

Reply:
Looks like the `axlsx` gem doesn't support reading Excel files, but there are [a few other gems](https://spin.atomicobject.com/2017/03/22/parsing-excel-files-ruby/).

You can get the file contents from `response.body`, if the gem only supports loading files you can use `Tempfile` from Ruby's standard library:

```ruby
excel_file = Tempfile.new
excel_file.write(response.body)
excel_file.close
Roo::Spreadsheet.open(excel_file.path)
```

The nice thing about `Tempfile` is that the file will be automatically deleted once the variable goes out of scope (i.e. after the test finishes)

Reply:
> Also I am not sure if it is needed

IMHO it's definitely low-priority :)

Reply:
yup agree - its not really priority, just came in my mind, when I noticed a cell was missing in the excel